docs/resources/control-flow/user-interactivity/forms/form-actions/reset-form-field.md

```diff
@@ -0,0 +1,31 @@
+---
+slug: reset-form-field
+title: Reset Form Field [Action]
+tags: []
+description: Learn how to add Reset Form Field action in your FlutterFlow app.
+sidebar_position: 1
+---
+# Reset Form Field [Action]
+
+This action allows you to reset a value for the Form widgets. This is helpful in clearing out any previously entered data and allows users to start fresh.
+
+For example, If a form has been successfully submitted, it may make sense to clear out the form fields so that the user can enter new data if they want to submit the form again.
+
+![Reset all fields for a medical appointment](../../imgs/reset-form-field-demo.gif)
+
+## Adding Reset Form Field action
+
+Follow the steps below to add this action to any widget.
+
+1. Select the **Widget** (e.g., Button, etc.) on which you want to add the action.
+2. Select **Actions** from the properties panel, If it's the first action, click **+ Add Action** button. Otherwise, click the "**+**" button below the previous action tile (inside *Action Flow Editor*) and select **Add Action**.
+3. Search the **Reset Form Field** (under *State Management*) action and select the widget you would like to reset.
+
+You can reset the same types of widgets in a single action only. e.g., multiple *TextField* widgets.
+
+![Adding Reset Form Field action](../../imgs/adding-reset-action.gif)
+
+:::info
+You can also reset form fields that are inside the components.
+![reset-form-field-component](../../imgs/reset-form-field-component.png)
```

Comment:
Add p tag before image

Reply:
Added, but seems like no effect to me, and adding <br> tag adds more gap.